rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] I would like to recommend you make sure that all your instance variables have `attr_accessor` since there are both getter and setter.  attr_reader defines only getter.  Please fix it to make your code work as expected :+1:

Reply:
The code has been refactored to use the `attr_accessor`.